source/stylesheets/refills/_accordion-tabs.scss

```diff
@@ -91,13 +82,13 @@
   .tab-content {
     background: $tab-content-background;
     display: none;
-    padding: $base-spacing $gutter;
+    padding: $tab-spacing $gutter;
     width: 100%;
 
     @include media($tab-mode) {
-      border-bottom-left-radius: $base-border-radius;
-      border-bottom-right-radius: $base-border-radius;
-      border-top-right-radius: $base-border-radius;
+      border-bottom-left-radius: $tab-border-radius;
```

Comment:
Properties should be ordered border, border-bottom-left-radius, border-bottom-right-radius, border-top-right-radius, float